bls/globals.go

```diff
@@ -152,20 +152,19 @@ func EvaluatePolyInEvaluationForm(yFr *Fr, poly []Fr, x *Fr, rootsOfUnity []Fr,
 	MulModFr(yFr, &y, &tmp)
 }
 
-func PolyLinComb(vectors [][]Fr, scalars []Fr) ([]Fr, error) {
+func PolyLinComb(vectors [][]Fr, scalars []Fr, polyDegree int) ([]Fr, error) {
```

Comment:
why not use `len(scalars)` as `polyDegree`?

Reply:
I modified this method to account for the case that we have no scalars and no vectors. I think len(scalars) == 0 in that case